ui/package-lock.json

```diff
@@ -17027,6 +17027,7 @@
         "node": ">=0.10.0"
       }
     },
+<<<<<<< HEAD
```

Comment:
package-lock.json was not merged correctly, otherwise this looks good.

Reply:
I just push a new changes to 14 lines of ui/package-lock.json to submenu2. How do I see it here?

Reply:
@DungLe2911 You can see it in the Files changed tab. The comment still shows up here because the conversation has not been resolved. Note the "Outdated" badge in yellow next to the file name above the diff. If that shows up, that means the file was changed enough that it can't be updated. Because this issue is fixed, you may resolve this conversation if you want to.

Reply:
This is very helpful for me to understand more on github. I really appriciate all the supports @colbycocking  @lgorence